canvas_modules/common-canvas/__tests__/_utils_/common-canvas-utils.js

```diff
@@ -46,7 +46,7 @@ export function createIntlCommonCanvas(
 	decorationActionHandler,
 	selectionChangeHandler,
 	tipHandler,
-
+	showBottomPanel,
 	toolbarConfig,
```

Comment:
In the calling code the 9th and 10th parameters are showBottomPanel and showRightFlyout but here they are not in the same order.